.github/workflows/deploy.yml

```
@@ -74,23 +75,7 @@ jobs:
         cf_password: ${{ secrets.CLOUDGOV_PASSWORD }}
         cf_org: gsa-tts-benefits-studio
         cf_space: notify-staging
-        push_arguments: >-
-          --vars-file deploy-config/staging.yml
-          --var DANGEROUS_SALT="$DANGEROUS_SALT"
-          --var SECRET_KEY="$SECRET_KEY"
-          --var ADMIN_CLIENT_USERNAME="notify-admin"
-          --var ADMIN_CLIENT_SECRET="$ADMIN_CLIENT_SECRET"
-          --var NEW_RELIC_LICENSE_KEY="$NEW_RELIC_LICENSE_KEY"
-          --var NR_BROWSER_KEY="$NR_BROWSER_KEY"
-          --var COMMIT_HASH="$COMMIT_HASH"
-          --var LOGIN_PEM="$LOGIN_PEM"
-          --var LOGIN_DOT_GOV_CLIENT_ID="$LOGIN_DOT_GOV_CLIENT_ID"
-          --var LOGIN_DOT_GOV_USER_INFO_URL="$LOGIN_DOT_GOV_USER_INFO_URL"
-          --var LOGIN_DOT_GOV_ACCESS_TOKEN_URL="$LOGIN_DOT_GOV_ACCESS_TOKEN_URL"
-          --var LOGIN_DOT_GOV_LOGOUT_URL="$LOGIN_DOT_GOV_LOGOUT_URL"
-          --var LOGIN_DOT_GOV_BASE_LOGOUT_URL="$LOGIN_DOT_GOV_BASE_LOGOUT_URL"
-          --var LOGIN_DOT_GOV_SIGNOUT_REDIRECT="$LOGIN_DOT_GOV_SIGNOUT_REDIRECT"
-          --var LOGIN_DOT_GOV_INITIAL_SIGNIN_URL="$LOGIN_DOT_GOV_INITIAL_SIGNIN_URL"
+        cf_command: "push -f manifest.yml --vars-file deploy-config/staging.yml --var var-name=${{ secrets.DANGEROUS_SALT }} --var var-name=${{ secrets.SECRET_KEY }} --var var-name=${{ ADMIN_CLIENT_USERNAME=admin-notify }} --var var-name=${{ secrets.ADMIN_CLIENT_SECRET }} --var var-name=${{ secrets.NEW_RELIC_LICENSE_KEY }} --var var-name=${{ secrets.NR_BROWSER_KEY }} --var var-name=${{ COMMIT_HASH }} --var var-name=${{ secrets.LOGIN_PEM }} --var var-name=${{ LOGIN_DOT_GOV_CLIENT_ID }} --var var-name=${{ LOGIN_DOT_GOV_USER_INFO_URL }} --var var-name=${{ LOGIN_DOT_GOV_ACCESS_TOKEN_URL }} --var var-name=${{ LOGIN_DOT_GOV_LOGOUT_URL }} --var var-name=${{ LOGIN_DOT_GOV_BASE_LOGOUT_URL }} --var var-name=${{ LOGIN_DOT_GOV_SIGNOUT_REDIRECT }} --var var-name=${{ LOGIN_DOT_GOV_INITIAL_SIGNIN_URL }} --strategy rolling"
```

D'oh!  We do need to do something different for the `ADMIN_CLIENT_USERNAME` here...

I think `--var ADMIN_CLIENT_USERNAME=admin-notify` will work here.  The rest should be okay!

```suggestion
        cf_command: "push -f manifest.yml --vars-file deploy-config/staging.yml --var var-name=${{ secrets.DANGEROUS_SALT }} --var var-name=${{ secrets.SECRET_KEY }} --var ADMIN_CLIENT_USERNAME=admin-notify --var var-name=${{ secrets.ADMIN_CLIENT_SECRET }} --var var-name=${{ secrets.NEW_RELIC_LICENSE_KEY }} --var var-name=${{ secrets.NR_BROWSER_KEY }} --var var-name=${{ COMMIT_HASH }} --var var-name=${{ secrets.LOGIN_PEM }} --var var-name=${{ LOGIN_DOT_GOV_CLIENT_ID }} --var var-name=${{ LOGIN_DOT_GOV_USER_INFO_URL }} --var var-name=${{ LOGIN_DOT_GOV_ACCESS_TOKEN_URL }} --var var-name=${{ LOGIN_DOT_GOV_LOGOUT_URL }} --var var-name=${{ LOGIN_DOT_GOV_BASE_LOGOUT_URL }} --var var-name=${{ LOGIN_DOT_GOV_SIGNOUT_REDIRECT }} --var var-name=${{ LOGIN_DOT_GOV_INITIAL_SIGNIN_URL }} --strategy rolling"
```